### ISSUE TITLE
React Scripts build including Elastic Search UI gives "Failed to parse source map" warnings for react-search-ui-views

### ISSUE THREAD

**Issue by ChrisSamo632** [elastic/search-ui#688]
**Describe the bug**
When including `@elastic/react-search-ui-views` version `1.10.1` in a Create React App using `react` version `17.0.2` and `react-scripts` version `5.0.0`, several "Failed to parse source map" warnings are emitted when executing `react-scripts build` for the application.

These warnings **didn't** appear when using Elastic Search UI `1.10.0` or lower.

**To Reproduce**

1. Create React App with stated dependency imports
2. `yarn install`
3. `react-scripts build`

**Expected behavior**
React Scripts build a production-ready application including the Elastic Search UI components **without** emitting the warnings (And with all required source mappings included).

**Screenshots**

```
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/_base.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/_base.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_boolean-facet.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_boolean-facet.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_error-boundary.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_error-boundary.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_facets.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_facets.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_multi-checkbox-facet.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_multi-checkbox-facet.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_paging-info.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_paging-info.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_paging.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_paging.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_result.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_result.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_results-container.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_results-container.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_results-per-page.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_results-per-page.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_search-box.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_search-box.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_select.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_select.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_single-links-facet.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_single-links-facet.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_sorting.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/components/_sorting.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/layouts/_layout.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/layouts/_layout.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/modules/_mixins.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/modules/_mixins.scss'
Failed to parse source map from '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/modules/_variables.scss' file: Error: ENOENT: no such file or directory, open '/app/node_modules/@elastic/react-search-ui-views/src/styles/themes/reference-ui/modules/_variables.scss'
```

**Which backends and packages are you using:**
Backend: Elasticsearch (7.17.1) although accessed through a custom Node/Express adapter
Packages: react-search-ui (1.10.1), react-search-ui-views (1.10.1)

**Comment by joemcelroy** [comment]
Thanks for reporting. In the minor release, we made changes to our styles transformation. My first thought is that its missing the scss files within the package. We will look into it soon on why our changes affected this

**Comment by ChrisSamo632** [comment]
> Thanks for reporting. In the minor release, we made changes to our styles transformation. My first thought is that its missing the scss files within the package. We will look into it soon on why our changes affected this

I wonder whether this same change has broken the styling around the `<Paging />`  component from `react-search-ui`, we now get the following in our UI:

<img width="80" alt="elastic_search-ui_paging" src="https://user-images.githubusercontent.com/12159006/160691058-920f6938-fb55-4c25-8b37-212b08b7d476.png">

**Comment by joemcelroy** [comment]
thank you for flagging that too! I can replicate the issue here. Will come back with a fix.

**Comment by joemcelroy** [comment]
now been released under v1.10.2. Tested in my own local demo app and works as expected. Hope you get the same results too!
